sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -527,4 +536,43 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
       assert(df.filter("_1 IS NOT NULL").count() === 4)
     }
   }
+
+  test("Fiters should be pushed down for vectorized Parquet reader at row group level") {
```

Comment:
We already have test for the pushed down filters for ParquetRecordReader. But it is at individual record level. If you mean row group level, because ParquetRecordReader doesn't expose a count for the row group, I think we can't check if the filter is pushed down. Besides, it seems to be the functionality of ParquetRecordReader, and I think it should be unit test in parquet project.